### ISSUE TITLE
get hidden state

### ISSUE THREAD

**Issue by fousdfrf** [abetlen/llama-cpp-python#1695]
In the process of inferring tokens, how to return the hidden state before the lm_head and the logits after the lm_head
